### DISCUSSION TITLE
My Account got suspended for no reason and it is causing me trouble

### DISCUSSION THREAD

**Discussion by testfalcon2024** [community/community#119956] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have created multiple threads and tickets but there is no response from the support team. I am working in a company and the company linked mail address's github account got suspended. This is stopping the development process and my other works are stuck. So please help me in this. 

Ticket ID: 2707049

**Comment by harshm2601** [comment]
Same Issue here , if you find any solution , kindly contact me

**Reply by testfalcon2024** [reply]
Yes the support is so worse ..

**Reply by Naman-Acharya** [reply]
I don't know how but most of users faced similar issue even my account got suspended without any reason. Might be this is because of malware as per the current news but still unaware about it. I hope we get our account back as soon as possible because specially most of my work was on Github and with Github Copilot and all the work was suddenly stopped because of this incident. 

My Ticket No: #2737022

**Reply by imseamoon** [reply]
I got the same issue , I might just get fired. If anyone got the solution please be sure to drop.

**Comment by MuhammadRayyan16** [comment]
Response #2

**Comment by ghost** [comment]
@testfalcon2024 
Did you get a response? Mine got suspended too, it has been more than a month and still no response.
ticket: #2719793
